tests/components/sighthound/test_image_processing.py

```diff
@@ -91,3 +95,23 @@ def capture_person_event(event):
     state = hass.states.get(VALID_ENTITY_ID)
     assert state.state == "2"
     assert len(person_events) == 2
+
+
+async def test_save_image(hass, mock_image, mock_detections):
+    """Save a processed image."""
+    VALID_CONFIG_SAVE_FILE = deepcopy(VALID_CONFIG)
```

Comment:
Use lowercase variable names for local variables.

Reply:
ok